manifests/Plex/PlexMediaServer/1.19.4.2935.yaml

```
@@ -0,0 +1,18 @@
+Id: plex.plexmediaserver
+Version: 1.19.4.2935
+Name: Plex Media Server
+Publisher: Plex
+License: Plex END USER LICENSE AGREEMENT
+LicenseUrl: https://www.plex.tv/media-server-downloads/#remodal-terms
+AppMoniker: PlexMediaServer
+Tags: PlexMediaServer
```

```suggestion
Tags: pms
```

'PlexMediaServer' doesn't have to be a tag since searching for that keyword will already bring up this package (matches on title and package name)